src/instrument.ts

```diff
@@ -45,12 +45,21 @@ interface SQLStatement {
   sql: string;
 }
 
-interface observabilityOptions {
+/*
+ * ObservabilityOptions defines the configuration with which
+ * startTrace may be invoked if set.
+ * @property [tracerProvider] is the injected TracerProvider to use,
+ * otherwise the global TracerProvider shall be used.
+ * @property [enableExtendedTracing] when set signifies that spans started
+ * with an accompanying an SQL statement shall be annotated with that SQL.
+ * Alternatively, you could set environment variable `SPANNER_ENABLE_EXTENDED_TRACING=true`.
```

Comment:
```suggestion
 * @property [enableExtendedTracing] Sets whether to enable extended OpenTelemetry tracing. Enabling this option will add the
     * following additional attributes to the traces that are generated by the client
     * db.statement: Contains the SQL statement that is being executed.
 * Alternatively, you could set environment variable `SPANNER_ENABLE_EXTENDED_TRACING=true`.
```

Reply:
Thank you @surbhigarg92! Isn't hard-coding the variable defeating the purpose of prose documentation for customers or are we more like pointing them to the actual variable to look for?